nautobot/docs/additional-features/relationships.md

```diff
@@ -0,0 +1,65 @@
+# Relationships
```

Comment:
This file would be better located as `nautobot/docs/models/extras/relationship.md`. That way it will be accessible as online help when creating or editing a Relationship record.

Also please add a link to this file in the appropriate section of `mkdocs.yml` so that it appears in the documentation sidebar. Thanks!

Reply:
Is that location in addition to this one or in place of?

Reply:
In place of, thanks. It's fine to link from the "Additional Features" subhead in `mkdocs.yml` to a file in `models/extras` - we already do that for several other features.

Reply:
Moved to the specified location and added the new location to mkdocs.yml